test/installer/scripts/omsadmin_systest.rb

```diff
@@ -296,4 +310,28 @@ def test_reconstruct_from_partial_empty_omsadmin_conf
                  "tmp directory was mistakenly created")
   end
 
+  def test_show_omsagent_proc_limit_default
+    set_omsagent_proc_limit()
+    lso = show_omsagent_proc_limit()
+    assert_match(/^-?\d+$/, lso, "The command to show what should be the default proc limit did not yield an integer.")
+  end
+
+  def test_show_omsagent_proc_limit_5000
+    set_omsagent_proc_limit(5000)
+    lso = show_omsagent_proc_limit()
+    assert_match(/^5000$/, lso, "Should have been 5000 process limit.")
```

Comment:
It would be more explicit to change this to `assert_equal("5000", lso, "Should have been 5000 process limit.")` since you know the exact string that should get printed out; same for `test_show_omsagent_proc_limit_two_entries`

Reply:
Yep.